app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/UIModel.scala

```diff
@@ -49,8 +49,8 @@ object APISubscriptions {
       val subscriptionGroups = subscriptions.groupBy(_.isTestSupport)
       val testApis           = subscriptionGroups.get(true).map(groupSubscriptionsByServiceName).getOrElse(Seq.empty).sortBy(_.apiHumanReadableAppName)
       val apis               = subscriptionGroups.get(false).map(groupSubscriptionsByServiceName).getOrElse(Seq.empty).sortBy(_.apiHumanReadableAppName)
-      val exampleApis        = apis.find(sub => sub.apiServiceName == EXAMPLE_API_NAME)
-      Some(GroupedSubscriptions(testApis, apis.filter(sub => sub.apiServiceName != EXAMPLE_API_NAME), exampleApis))
+      val exampleApis        = apis.find(sub => sub.apiServiceName.value == EXAMPLE_API_NAME)
```

Comment:
I'd make the constant a ServiceName and skip .value

Reply:
Fixed